fendermint/app/src/cmd/key.rs

```diff
@@ -66,6 +70,27 @@ cmd! {
   }
 }
 
+cmd! {
+  AddPeer(self) {
+    let node_key = NodeKey::load_json_file(&self.node_key_file).unwrap();
```

Comment:
The [exec](https://github.com/consensus-shipyard/fendermint/blob/60ac4f47331f5c95a38b00e2d0aecace0a903caf/fendermint/app/src/cmd/mod.rs#L43) method returns `anyhow::Result`, which is [handled](https://github.com/consensus-shipyard/fendermint/blob/60ac4f47331f5c95a38b00e2d0aecace0a903caf/fendermint/app/src/main.rs#L29-L31) in `main` by logging and exiting. 

So let's not use `.unwrap()` inside but the `?` operator [like all the other commands](https://github.com/consensus-shipyard/fendermint/blob/60ac4f47331f5c95a38b00e2d0aecace0a903caf/fendermint/app/src/cmd/key.rs#L46), perhaps with [context](https://docs.rs/anyhow/latest/anyhow/trait.Context.html) to attach some meaningful error message.

Reply:
Fixed.